docs/tutorials/zero-to-hero.md

```diff
@@ -41,130 +32,35 @@ Let's get started!
-
-  if(result) {
-    return result;
-  }
-
-  return "";
-}
-```
-
-### Wiring the functions
-
-Great! Now that we've implemented these functions, let's ensure they're available in the window object.
-
-To use these functions, we need to "attach" them to the contract object made available by our JavaScript library `near-api-js`.  This is easily done by adding their name (as a string) to an collection of available contract methods.
-
-NEAR contracts support two types of methods:
-- `view` methods to read from the blockchain without changing state
-- `change` methods to alter the state of the blockchain
+## Step 1: Logging in with NEAR
+### Update the front end
 
-We have one of each.
-
-> In the file `src/main.js`
-> - Replace the values of `viewMethods` and `changeMethods` with our new smart contract methods.
-
-```js
- // Initializing our contract APIs by contract name and configuration
-  const contract = await new nearlib.Contract(walletConnection.account(), nearConfig.contractName, {
-    // View methods are read-only – they don't modify the state, but usually return some value
-    viewMethods: ['getResponse'],   // <-- find this line and change it to match
-    // Change methods can modify the state, but you don't receive the returned value when called
-    changeMethods: ['setResponse'],  // <-- find this line and change it to match
-    // Sender is the account ID to initialize transactions.
-    sender: accountId
-  })d
-```
-
-The edits we just made will allow us to invoke these methods on the contract object using basic JavaScript object syntax like this `contract.setResponse("hello world")` and `contract.getResponse()`.
-
-<blockquote class="info">
-<strong>did you know?</strong><br><br>
-
-We've just exposed a NEAR smart contract to our JavaScript client side code.
-
-Once deployed (this happens automagically in NEAR Studio when we click Run) the smart contract will be compiled to Wasm and deployed to the NEAR blockchain.   We can then invoke its methods using JavaScript which you will do in a few moments.
-
-If you're curious about how this works under the hood, have a look at the [`near-api-js` source code](https://github.com/near/near-api-js/blob/master/src/contract.ts).  You'll find we're just wrapping a couple of layers of abstraction around our RPC interface.
-
-</blockquote>
-
-### Cleaning up the front end
-
-Now let's clean up the front end.
+Let's change the front end so that it makes sense for our use case. Eventually we want to display the bitcoin price on the page, so let's update everything accordingly.
 
 > In the file `src/index.html`
-> - Replace the contents of `div.jumbotron` (the div with **class**="jumbotron") with the following code
-
-```html
-<h3>Zero to Hero Tutorial</h3>
-<strong>Set and get a response from the blockchain</strong>
+> - Replace the header "Who was the last person to say hi" with the code `<h1>Near oracle demo</h1>`. If you would like to add any other HTML changes, this is where to do this.
```

Comment:
Let's tell them to use the [Wallet Example](https://github.com/near-examples/wallet-example/blob/54e682a79fb286432c873bad971f69fd161d3411/src/index.html) where "Who was the last person" is mentioned, if we're going to instruct them to do this. Also, we didn't list out `src/index.html` around line 43, which we probably should.

Reply:
When I replaced the entire header, which on Wallet Example, is:
`<h1><a id="who" target="_blank">Who</a> was the last person to say hi?</h1>`
and replaced it with:
`<h1>Near oracle demo</h1>`
then a few steps later it's going to try to find `#who` and replace it and not find it.
(Search for "Once signed in with NEAR you will see "Hi, YOUR ACCOUNT!")
So we need to either tell them not to cut out the "Who" part, or cut out the above expectation to see "Hi, YOUR ACCOUNT"
I'll stop here for now, I think this needs to get run through with fresh eyes, step by step. Happy to rereview after that

Reply:
done